Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    fastlane (2.171.0)
+    fastlane (2.172.0)
```

Comment:
This should be updated to 2.173.0 by "bundle install" when merging this PR😅

Reply:
This shouldn't be pushed into PRs doesn't it? It gets updated in the release process afaik

Reply:
Hmm I haven't thought when this should get updated🙁 `Gemfile.lock` started to be checked in from this PR https://github.com/fastlane/fastlane/pull/17892 

I didn't intend to have it work like that way but it sounds fair to me!! For now I'll discard this change and then follow it up in another PR 👍 Thank you for your review.

Reply:
Ahhhh, I was wondering when Gemfile.lcok was starting to get committed! I totally didn’t see this in PR #17892. This has been messing up the release process a little bit but now I totally understand where it started coming from and how to fix 😊